build/ci/.azure-devops-docs.yml

```diff
@@ -25,10 +25,9 @@ jobs:
     parameters:
       nugetPackages: $(NUGET_PACKAGES)
 
-  - template: templates/dotnet-install.yml
```

Comment:
Is the goal to remove this template from the steps?

Reply:
Nope, not really a goal. But this one here is a duplicate. We are already using `dotnet-mobile-install-windows.yml` which contains `dotnet-install.yml`